### PR TITLE
chore: export synthetic admin key updates for all undeleted nodes

### LINKED ISSUE #16937: (0.57) Publish synthetic `NodeUpdate`s at upgrade boundary to supplement `0.0.102` for mirror nodes
### Problem

Since the `NodeAddressBook` message doesn't have admin key information, we would like the initial admin keys to be explicitly published to mirror nodes in release 0.57 via synthetic `NodeUpdate` transactions.

### Solution

Finish `SystemSetup.doPostUpgradeSetup()` on branch `release/0.57` by publishing synthetic `NodeUpdate`s for each node that do nothing except "update" the admin keys to the existing keys.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Closes #16937

### CODE REVIEW

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SystemSetup.java [hunk 1]
```diff
@@ -194,6 +200,25 @@ ctx, createFileID(filesConfig.hapiPermissions(), config), bytes),
         if (autoNodeAdminKeyUpdates.tryIfPresent(adminConfig.upgradeSysFilesLoc(), systemContext)) {
             dispatch.stack().commitFullStack();
         }
+        // In 0.57 specifically, we export synthetic node updates for all non-deleted nodes in the address book,
+        // exposing their admin keys to mirror nodes to supplement the NodeAddressBook message in 0.0.112
```
Comment (steven-sheehy) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SystemSetup.java:204]: Does this then need to be removed after 0.57 so it doesn't continually send synthetic node updates in every future release? Should it be moved to a migration instead?

Comment (tinker-michaelj) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SystemSetup.java:204]: Hmm @steven-sheehy not sure I understand; this is being added just in `release/0.57` not `develop`.

Comment (steven-sheehy) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SystemSetup.java:204]: Nevermind me then. I'm used to changes always going to main and cherry picked to a release branch.

**Review by iwsimon — APPROVED** [review]
LGTM, thanks @tinker-michaelj

**Pull request merged**